x/emissions/types/params.go

```diff
@@ -125,6 +129,19 @@ func validateBallotMaturityBlocks(i interface{}) error {
 	return nil
 }
 
+func validatePendingBallotsBufferBlocks(i interface{}) error {
+	v, ok := i.(int64)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	// pending ballots buffer blocks can be 0 to delete pending ballots immediately
+	if v < 0 {
```

Comment:
We should prevent using 0 as well

Reply:
okay changed to 1 .
But 0 would allow us to replicate the current behaviour if we ever need to